projects/plugins/jetpack/3rd-party/jetpack-videopress.php

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
I have the same comments in the jetpack-protect.php file in this one, I won't messy the PR by copying them